signingscript/requirements/base.in

```diff
@@ -8,5 +8,5 @@ scriptworker
 # version 39.x.x
 taskcluster<40
```

Comment:
We can unpin this now, but doesn't have to be in this PR.

Reply:
may as well do it, it's easy :)